pyproject.toml

```diff
@@ -10,8 +10,8 @@ alembic = "^1.7.6"
 Authlib = "^0.15.5"
 bcrypt = "^3.2.0"
 boto3 = "^1.26"
-cpr-data-access = {git = "https://github.com/climatepolicyradar/data-access.git", tag = "v0.2.9"}
-fastapi = "^0.89.0"
+cpr-data-access = {git = "https://github.com/climatepolicyradar/data-access.git", rev = "4ec055db7cd93f2c9b275c5a6fd90145c6aa6851", extras = ["vespa"]}
```

Comment:
Why are we using `rev` and not `tag`?

Reply:
The commit hasn't actually been merged yet, its part of: https://github.com/climatepolicyradar/data-access/pull/105 We wanted to test that the change in that PR worked here, I'd want it properly merged and switched here to a tag before we update prod though

Reply:
No problem, I'll tag first